networkit/cpp/graph/test/GraphGTest.cpp

```diff
@@ -2007,19 +1992,79 @@ TEST_P(GraphGTest, testForWeightedEdgesWithIds) {
         } else {
             ASSERT_EQ(3.0, sum);
         }
+    }
+}
+
+TEST_P(GraphGTest, testParallelForEdgesWithIds) {
+    std::vector<Graph> graphs;
+    graphs.push_back(Graph(10, false, false));
+    graphs.push_back(Graph(10, false, true));
+    graphs.push_back(Graph(10, true, false));
+    graphs.push_back(Graph(10, true, true));
```

Comment:
I suggest to use [emplace_back](https://en.cppreference.com/w/cpp/container/vector/emplace_back)